Formula/geocouch.rb

```diff
@@ -137,4 +139,6 @@ def caveats; <<-EOS.undent
     run 'brew info geocouch'.)
     EOS
   end
+
+  test {}
```

Comment:
Thank you for the response. So, what's the preferred action plan for now? Should I enhance this now or wait until it's merged into couchdb formula?